package.json

```diff
@@ -20,7 +20,8 @@
         "check:lint": "eslint --color .",
         "check:types": "ts-node scripts/check-for-typescript.ts tsc --noEmit --pretty",
         "check:license": "ts-node scripts/nrfconnect-license.ts check",
-        "prepare": "husky install"
+        "prepare": "husky install",
+        "update-publish-script": "tsc scripts/nordic-publish.ts -t es2015 -m node16"
```

Comment:
We could probably also use `swc` or `esbuild` instead of `tsc`. But this compilation is so fast and done so rarely that it is fine to just use `tsc`.